src/PMS/PMS.cpp

```diff
@@ -147,6 +148,36 @@ void PMSBase::handle() {
  */
 bool PMSBase::isFailed(void) { return failed; }
 
+/**
+ * @brief Increate number of fail
```

Comment:
Increase vs. update. However, we can remove such comments they don't provide any value.